e2e-tests/tests/headlampPage.ts

```diff
@@ -74,8 +84,8 @@ export class HeadlampPage {
     await this.page.click('a.MuiMenuItem-root:has-text("Log out")');
     await this.page.waitForLoadState('load');
 
-    // Expects the URL to contain c/main/token
-    await this.hasURLContaining(/.*token/);
+    // Expects the URL to contain c/test/token
+    // await this.hasURLContaining(/.*token/);
```

Comment:
I guess this doesn't work now? Do you know why this doesn't work?
Is it because there are two clusters?

I think it's fine to leave it commented out if it's not working and it's too hard to fix.

Reply:
> I guess this doesn't work now? Do you know why this doesn't work?
> Is it because there are two clusters?

Yes it isn't working. When I was testing this multi-cluster setup on my local machine, on logout, this expected URL was not appearing in the browser. I am not really sure about why this might be happening so can't really comment on the underlying reason :(.

Reply:
Cool. Thanks for the explanation.

(mainly I wanted to make sure it wasn't commented out by accident)